src/main/steps/applicant1/connection-summary/content.ts

```diff
@@ -6,14 +6,14 @@ import { CommonContent } from '../../common/common.content';
 export const jurisdictionMoreDetailsContent = (
   connections: JurisdictionConnections[] | undefined,
   isDivorce: boolean,
-  isRespondent = false
+  showAllResidences = false
 ): { connectedToEnglandWales: string; readMore: string } => {
   const habConnection = enContainsHabitualResConnection(connections);
   const domConnection = enContainsDomConnection(connections);
   const residualConnection = enContainsResidualConnection(connections, isDivorce);
 
   const connectionIndex =
-    isRespondent || (habConnection && domConnection && residualConnection)
```

Comment:
do you not need the isRespondent check anymore?

Reply:
isRespondent is only there to decide if to show all the residences.  I have changed it to showAllResidences.  It is a boolean field

Reply:
In some places we show all the residences content regardless of whether you're a respondent or not.  The name isRespondent is confusing